packages/reports/src/Components/placeholder-component.jsx

```diff
@@ -24,7 +24,7 @@ PlaceholderComponent.propTypes = {
     empty_message_component: PropTypes.func,
     has_selected_date: PropTypes.bool,
     is_empty: PropTypes.bool,
-    is_loading: PropTypes.node,
+    is_loading: PropTypes.bool,
```

Comment:
eliminating a console warning: _Failed prop type: Invalid prop `is_loading` supplied to `PlaceholderComponent`, expected a ReactNode._